src/core/db/repository/category.py

```diff
@@ -15,11 +15,11 @@ def __init__(self, session: AsyncSession = Depends(get_session)) -> None:
         super().__init__(session, Category)
 
     async def get_unarchived_parents(self) -> list[Category]:
-        categories = await self._session.execute(
-            select(Category).where(and_(not_(Category.archive)), Category.parent_id.is_(None)))
-        return categories.scalars().all()
+        categories = await self._session.scalars(
+            select(Category).where(Category.is_archived == False).where(Category.parent_id.is_(None)))
+        return categories
```

Comment:
Я бы сделал как-то так, если согласен, можешь немного отрефачить. Ради такой предирки ПР не охото задерживать)

```suggestion
        return await self._session.scalars(
            select(Category)
            .where(Category.is_archived == False)
            .where(Category.parent_id == None)  # noqa
        )
```

Reply:
это изначально было так написано, я не трогал. поменяю